code/modules/keybindings/bindings_atom.dm

```diff
@@ -16,3 +16,6 @@
 		if((movement_dir & EAST) && (movement_dir & WEST))
 			movement_dir &= ~(EAST|WEST)
 		user.Move(get_step(src, movement_dir), movement_dir)
+		return !!movement_dir //true if there was actually any player input
```

Comment:
Not not? Wouldn't that just be movement_dir?
I see what they're doing by casting it into a boolean, but it's very weird since a true/false returns true on any non-false/0